modules/xml/src/main/java/org/mule/module/xml/transformer/XsltTransformer.java

```diff
@@ -456,6 +482,17 @@ public void setMaxIdleTransformers(int maxIdleTransformers)
         transformerPool.setMaxIdle(maxIdleTransformers);
     }
 
+    public long getTimeBetweenTransformerEvictions()
+    {
+        long evictionTime = transformerPool.getTimeBetweenEvictionRunsMillis();
+        return evictionTime;
```

Comment:
This should be a single line. The variable names are the same as the methods that are called. evictionTime alone is confusing also.